.vscode/settings.json

```diff
@@ -17,5 +17,8 @@
     "temp": true,
     "**/test/**/temp": false,
     "coverage": true
+  },
+  "files.associations": {
+    "rush.json": "jsonc",
```

Comment:
I can remove this if necessary. I added it so that I could read through the `rush.json` file without the red squiggles.

Reply:
This is what we recommend [in the docs](https://api-extractor.com/pages/setup/invoking/#3-create-a-template-config-file):

```json
"files.associations": { "*.json": "jsonc" }
```

Reply:
This would mistakenly treat *every* JSON file as JSONC, which could result in adding invalid syntax in a regular JSON file. I can remove the edit, but I generally prefer to be more explicit with these kinds of associations.